sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamSortRelTest.java

```diff
@@ -49,16 +50,16 @@ public void prepare() {
             Types.DOUBLE, "price",
             Types.TIMESTAMP, "order_time"
         ).addRows(
-            1L, 2, 1.0, new Date(0),
-            1L, 1, 2.0, new Date(1),
-            2L, 4, 3.0, new Date(2),
-            2L, 1, 4.0, new Date(3),
-            5L, 5, 5.0, new Date(4),
-            6L, 6, 6.0, new Date(5),
-            7L, 7, 7.0, new Date(6),
-            8L, 8888, 8.0, new Date(7),
-            8L, 999, 9.0, new Date(8),
-            10L, 100, 10.0, new Date(9)
+            1L, 2, 1.0, new Timestamp(0),
```

Comment:
can you check if we've test case to cover `Types.DATE`?

Reply:
We don't. SoryByTimestamp() is the only test case that calls "order by order_time". Previously, TIMESTAMP is not supported so we used Date type here. If we want to test SortByDate(), we can extend current table with a fifth column named "order_date", just for testing purpose. What do you think? 

PS. The build has some problem. I will update the PR soon.